jenkins/janitor.py

```diff
@@ -50,34 +55,44 @@ def collect(project, age, resource, filt):
         resource: Definition of a type of gcloud resource.
         filt: Filter clause for gcloud list command.
     Returns:
-        A dict of condition : list of resource.
+        A dict of condition : list of gcloud resource object.
+    Raises:
+        ValueError if json result from gcloud is invalid.
     """
 
     col = collections.defaultdict(list)
 
     for item in json.loads(subprocess.check_output([
             'gcloud', 'compute', '-q',
             resource.name, 'list',
-            '--format=json(name,creationTimestamp.date(tz=UTC),zone,region)',
+            '--format=json(name,creationTimestamp.date(tz=UTC),zone,region,MANAGED)',
             '--filter=%s' % filt,
             '--project=%s' % project])):
 
-        if not item['name'] or not item['creationTimestamp']:
-            print '[Warning] - Skip item without name or timestamp'
-            print '%s' % item
-            continue
-        if resource.condition and not item[resource.condition]:
-            print '[Warning] - condition specified but not found'
-            continue
+        print '%s' % item
```

Comment:
nit: consider refactoring this to use logging instead of print (fine to punt to a future pr)